### PR TITLE
Extend e2e test for `metadata viewer` role

### PR DESCRIPTION
**Pull request by thanhz**

### CODE REVIEW

File: src/test/resources/features/DownloadMetadata.feature [hunk 1]
```diff
@@ -33,4 +33,30 @@ Feature: Download Metadata page
...
+    And the file checks are complete
+    And the user has created additional metadata
+    And an existing metadata review is in progress
+    Then the standard user logs out
+    Given A logged in transfer adviser user
+    When the logged in user navigates to the metadata review page
+    Then the user will be on a page with the label "1. Download and review transfer metadata"
+    And the user clicks the download metadata link
+
+  Scenario: The metadata viewer should be able to download the metadata excel
```
Comment (vimleshtna) [inline src/test/resources/features/DownloadMetadata.feature:51]: You can add examples to run the same tests with different inputs:

  ```
Scenario Outline: The <user> should be able to download the metadata excel
    Given A logged in standard user
    And an existing standard consignment for transferring body MOCK1
    When the logged in user navigates to the Download Metadata page
    And the file checks are complete
    And the user has created additional metadata
    And an existing metadata review is in progress
    Then the standard user logs out
    Given A logged in <user> user
    When the logged in user navigates to the metadata review page
    Then the user will be on a page with the label "1. Download and review transfer metadata"
    And the user clicks the download metadata link

    Examples:
      | user             |
      | transfer adviser |
      | metadata viewer  |
```

File: src/test/resources/features/MetadataReview.feature [hunk 2]
```diff
@@ -23,7 +23,7 @@ Feature: Metadata review pages
     When the logged in user navigates to the review-progress page
     Then the user will see the alert You can now complete your transfer
 
-  Scenario: Metadata review requested by user is rejected by DTA reviewer
+  Scenario: Metadata review requested by user is rejected by transfer adviser
```
Comment (vimleshtna) [inline src/test/resources/features/MetadataReview.feature:26]: Same as above. Use the `Examples`   with the `Scenario` to run the same test for both types of users.

**Review by vimleshtna — COMMENTED** [review]
Looks good. Just some minor comments. Thanks

File: src/test/resources/features/DownloadMetadata.feature (hunk 1 above)
Comment (thanhz) [reply src/test/resources/features/DownloadMetadata.feature]: That is really cool :open_mouth: I didn't know about this! I've made the changes

File: src/test/resources/features/MetadataReview.feature (hunk 2 above)
Comment (thanhz) [reply src/test/resources/features/MetadataReview.feature]: Made the recommended changes

**Review by vimleshtna — APPROVED** [review]
Looks good. Thanks

**Pull request merged**